examples/server-example.py

```diff
@@ -74,11 +72,26 @@ def multiply(parent, x, y):
     uri = "http://examples.freeopcua.github.io"
     idx = server.register_namespace(uri)
 
-    # get Objects node, this is where we should put our custom stuff
+
+    # create a new node type we can instantiate in our address space
+    base_type = server.get_root_node().get_child(["0:Types", "0:ObjectTypes", "0:BaseObjectType"])
```

Comment:
Looks like I did :-)
Committing it tomorrow. It is just a class with common nodes as members

On Wed, Jul 20, 2016, 21:25 Andrew <notifications@github.com> wrote:

> In examples/server-example.py
> <https://github.com/FreeOpcUa/python-opcua/pull/244#discussion_r71590241>:
>
> > @@ -74,11 +72,26 @@ def multiply(parent, x, y):
> >      uri = "http://examples.freeopcua.github.io"
> >      idx = server.register_namespace(uri)
> >
> > -    # get Objects node, this is where we should put our custom stuff
> > +
> > +    # create a new node type we can instantiate in our address space
> > +    base_type = server.get_root_node().get_child(["0:Types", "0:ObjectTypes", "0:BaseObjectType"])
>
> Looks interesting. I like it that way, but we need to make sure everyone
> else does.
>
> Did you forget to commit shortcuts.py?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/FreeOpcUa/python-opcua/pull/244/files/88f2ae0f075c2248c3421c956ab06051cd3d1ec9..785f185239fb050ac90cca77296cff6d996c58ee#r71590241>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACcfzqcuVWbf-OWgWu6vvsIr6_KXEYbMks5qXnYbgaJpZM4JLHjh>
> .
>